blog/2023/11/important-community-update-klaws-latest-actions-to-enhance-transparency.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: important-community-update-klaws-latest-actions-to-enhance-transparency
+title: Important community update Klaw's latest actions to enhance transparency
+authors: klawteam
+tags: [2.6.1, 2.5.2, transparency]
+date: Nov 22, 2023
+---
+
+## Important community update: Klaw's latest actions to enhance transparency
+
+<!-- vale off -->
+
+We, as an integral part of the Klaw open-source project, are committed to upholding the true spirit of open-source values. To ensure transparency and foster trust within our community, we have an important update to share.
+
+Discovery of legacy tracking code:
+We recently identified a piece of legacy, non-functional tracking code within our codebase. This code, originating from a previous version of our project, used Google tag (gtag) for analytics purposes. It was designed to track various user interactions on our site, including page views, scrolls, outbound link clicks, file downloads, video plays, site searches, and form interactions.
+
+Scope and deactivation of the tracker:
+This tracking code was specifically configured to gather data from kafkawize.com. We want to reassure our community that this means that no data from any other host has been or is being collected, accepted or stored by this code. As of February 2022, this configuration has been completely removed.
+
+Impact on users:
+We are pleased to inform you that the legacy code has not affected our newer Coral React UI. This interface, representing the latest development of our platform, remains free from any impact of this outdated tracking code.
+
+In commitment to our community's transparency, we are releasing software updates 2.5.2 and 2.6.1. These versions aim to completely remove this legacy code, allowing our users to transition to a version where their privacy and trust are further safeguarded.
+
+Your feedback is important. Stay updated and reach out to us via [GitHub/Email] for any queries or suggestions.
```

Comment:
``via [GitHub/Email]``

I just noticed this. Can we update this with a link to the GH repo?

Reply:
yes